torch/_dynamo/variables/ctx_manager.py

```diff
@@ -1185,9 +1185,9 @@ class StreamVariable(VariableTracker):
     def __init__(self, proxy, value, device, **kwargs) -> None:
         if proxy is not None and "example_value" in proxy.node.meta:
             assert proxy.node.meta["example_value"] == value
-        assert value.device.type == device.type, (
-            "stream value is not equal to the passed device"
-        )
+        assert (
```

Comment:
lol, unclear why this got formatted by lintrunner. Ignore this bit and I'll revert before merging.